dimod/binary_quadratic_model.py

```diff
@@ -205,2361 +193,244 @@ class for the K4 complete graph, where the nodes have biases
-        the Ising model subclass is instantiated upon the first use of the
-        :attr:`.spin` property and used in any subsequent reads.
+        Returns:
+            :class:`.BinaryQuadraticModel`
 
         Examples:
-            This example creates a QUBO model and uses the :attr:`.spin` property
-            to instantiate the corresponding Ising model.
-
-            >>> import dimod
-            ...
-            >>> bqm_qubo = dimod.BinaryQuadraticModel({0: -1, 1: -1}, {(0, 1): 2}, 0.0, dimod.BINARY)
-            >>> bqm_spin = bqm_qubo.spin
-            >>> bqm_spin   # doctest: +SKIP
-            BinaryQuadraticModel({0: 0.0, 1: 0.0}, {(0, 1): 0.5}, -0.5, Vartype.SPIN)
-            >>> bqm_spin.spin is bqm_spin
-            True
 
-        Note:
-            Methods like :meth:`.add_variable`, :meth:`.add_variables_from`,
-            :meth:`.add_interaction`, etc. should only be used on the base model.
+            >>> bqm = dimod.BinaryQuadraticModel({1: 1, 2: 2}, {(1, 2): 0.5}, 0.5, dimod.SPIN)
+            >>> bqm2 = bqm.copy()
 
         """
-        # NB: The existence of the _spin property implies that it is up to date, methods that
-        # invalidate it will erase the property
-        try:
-            spin = self._spin
-            if spin is not None:
-                return spin
-        except AttributeError:
-            pass
+        new = super().copy()
+        new.info.update(self.info)
+        return new
 
-        if self.vartype is Vartype.SPIN:
-            self._spin = spin = self
-        else:
-            self._counterpart = self._spin = spin = self.change_vartype(Vartype.SPIN, inplace=False)
+    def to_serializable(self, use_bytes=False, bias_dtype=np.float32,
+                        bytes_type=bytes):
+        """Convert the binary quadratic model to a serializable object.
 
-            # we also want to go ahead and set spin.binary to refer back to self
-            spin._binary = self
+        Args:
+            use_bytes (bool, optional, default=False):
+                If True, a compact representation representing the biases as
+                bytes is used. Uses :func:`~numpy.ndarray.tobytes`.
 
-        return spin
+            bias_dtype (data-type, optional, default=numpy.float32):
+                If `use_bytes` is True, this :class:`~numpy.dtype` will be used
+                to represent the bias values in the serialized format.
 
-    @property
-    def binary(self):
-        """:class:`.BinaryQuadraticModel`: An instance of the QUBO model subclass of
-        the :class:`.BinaryQuadraticModel` superclass (a BQM with binary variables).
+            bytes_type (class, optional, default=bytes):
+                This class will be used to wrap the bytes objects in the
+                serialization if `use_bytes` is true. Useful for when using
+                Python 2 and using BSON encoding, which will not accept the raw
```

Comment:
--> "If `use_bytes` is true, this class is used to wrap the bytes objects in the serialization" to save users time. I mention it because of the "Useful for when using" typo. 
BTW, "when using Python 2 and ", why are we still support Py2?